test/browser.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+const isWebWorker = require('detect-webworker')
+
+if (isWebWorker) {
+  // https://github.com/Joris-van-der-Wel/karma-mocha-webworker/issues/4
+  global.MFS_DISABLE_CONCURRENCY = true
+}
```

Comment:
Since there is a requirement to either set this variable or do some setup on the main thread (for webworkers) we should mention it and [link to it](https://github.com/ipfs/js-ipfs-mfs#a-note-on-concurrency) from the js-ipfs README so that it's easily discoverable and we don't get issues opened about it here.

Also a separate PR to add a examples for using MFS in webworkers with/without concurrency is needed and would be very 🚀 🎸 ✨ 💖

Reply:
I've added a paragraph to the README, will create an example in a separate PR.

Reply:
What does MFS with concurrency mean?

Reply:
nvm. Read the paragraph.